### PR TITLE
Move some files to the correct source location and rename a few other files.

### PR DESCRIPTION
**Pull request by jonwil**

### CODE REVIEW

**Comment by jonwil** [comment]
This is only a simple move/rename of files so will have no impact on functionality.

File: src/platform/w3dengine/client/w3ddisplay.cpp [hunk 1]
```diff
@@ -40,7 +40,7 @@
 #include "network.h"
 #include "particlesysmanager.h"
 #include "physicsupdate.h"
-#include "renderdevicedesc.h"
+#include "rddesc.h"
```
Comment (xezon) [inline src/platform/w3dengine/client/w3ddisplay.cpp:43]: Perhaps leave `renderdevicedesc` as is? `rddesc` is not more descriptive.

File: src/platform/w3dengine/client/drawable/draw/w3dprojectilestreamdraw.h
Comment (xezon) [inline src/platform/w3dengine/client/drawable/draw/w3dprojectilestreamdraw.h:1]: Whats the point of drawable/draw when drawable folder is empty?

**Review by xezon — COMMENTED** [review]
What about utf8string.h to asciistring.h and utf16string.h to unicodestring.h?

File: src/platform/w3dengine/client/drawable/draw/w3dprojectilestreamdraw.h
Comment (jonwil) [reply src/platform/w3dengine/client/drawable/draw/w3dprojectilestreamdraw.h:1]: I want to stick to the original filenames and paths where we know what they are. drawable probably contained files we don't have yet.

File: src/platform/w3dengine/client/w3ddisplay.cpp (hunk 1 above)
Comment (jonwil) [reply src/platform/w3dengine/client/w3ddisplay.cpp:43]: I want to stick to the original filenames and paths where we know what they are.

**Review by xezon — APPROVED** [review]

**Pull request merged**